test/controllers/IneligibleControllerSpec.scala

```diff
@@ -330,7 +330,7 @@ class IneligibleControllerSpec extends ItSpec {
                   "You must <a href=\"https://www.gov.uk/tell-hmrc-change-of-details\" class=\"govuk-link\">update your details with HMRC</a>. After you’ve updated your details, wait 3 working days before trying again online.")
               }
 
-              val expectedCallUsContent = "Call us on <strong>0300 322 7835</strong> as you may be able to set up a plan over the phone."
+              val expectedCallUsContent = "Call us on <strong>0300 123 1813</strong> as you may be able to set up a plan over the phone."
```

Comment:
is the SIMP RLS ineligible case being tested in this spec? It may need adding in - I'd expect you to have to change the expected text here based on the tax. regime